charts/dragonfly-operator/templates/deployment.yaml

```diff
@@ -43,6 +43,9 @@ spec:
             - --upstream=http://127.0.0.1:8080/
             - --logtostderr=true
             - --v=0
+            {{- with .Values.rbacProxy.extraArgs}}
```

Comment:
Just trying to understand why would you want to pass more args to the controller and the rbac proxy itself? (I can see the reason for Dragonfly args and we already have it.)

Reply:
Hi,

For the manager, this change would allow to set flags such as `-zap-log-level` and `-zap-encoder` for instance, which is not currently feasible if I'm right. For the rbac proxy there are many available flags that could be tuned by any user I assume.